inf-clojure.el

```diff
@@ -982,7 +983,9 @@ See variable `inf-clojure-arglists-form'."
     (cond
      ((null arglists-data) nil)
      ((stringp arglists-data) arglists-data)
-     ((listp arglists-data) arglists-result))))
```

Comment:
I don't remember why I was returning the result here, it looks like I should return the read data...Uhm..